pandas/io/formats/style.py

```diff
@@ -3139,3 +3150,37 @@ def css_calc(x, left: float, right: float, align: str):
             index=data.index,
             columns=data.columns,
         )
+
+
+def _refactor_levels(
```

Comment:
can do after, but this is fairly generic and doens't belong here; we also do this elsewhere in the codebase. pls create an issue / PR if you can.

Reply:
#42933